package.json

```diff
@@ -24,16 +24,18 @@
     "parse"
   ],
   "dependencies": {
-    "extend": "3.0.0",
-    "mr-parser": "0.2.1"
+    "extend": "^3.0.2",
+    "mr-parser": "^0.2.1"
   },
   "devDependencies": {
-    "coveralls": "2.11.4",
-    "doctoc": "0.14.2",
-    "mocha": "2.3.3",
-    "nodemon": "1.7.1",
+    "coveralls": "^2.11.4",
```

Comment:
I removed a few dependencies in https://github.com/mauriciopoppe/math-codegen/pull/8/files